Listenbrainz/UI/Screens/Yim/Charts/ChartsTitleView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ChartsTitleView.swift
+//  Listenbrainz
+//
+//  Created by Gaurav Bhardwaj on 11/01/24.
+//
+
+import SwiftUI
+
+struct ChartTitleView: View {
+  @ObservedObject var viewModel: YIMViewModel
+
+   init(viewModel: YIMViewModel) {
+       self.viewModel = viewModel
+   }
+  var body: some View {
+
+    ZStack {
+      Rectangle()
+        .foregroundColor(.clear)
+        .frame(height: UIScreen.main.bounds.height)
+        .background(Color.yimGreen)
+      VStack{
+        TopYimView(viewModel: viewModel)
+          .padding(.top,35)
+
+        Spacer()
+        Spacer()
```

Comment:
You don't need multiple spacers.

Reply:
@akshaaatt I used multiple spacers to place components in a fixed ratio.

Reply:
@gauravbhardwaj7 the spacer expands and takes up as much space between the elements, so having multiple spacers shouldn't make a difference.

Reply:
<img width="1440" alt="Screenshot 2024-01-13 at 11 17 13 PM" src="https://github.com/metabrainz/listenbrainz-ios/assets/106892217/7503504e-72a6-4281-b26d-24bd2da48417">
<img width="1440" alt="Screenshot 2024-01-13 at 11 17 00 PM" src="https://github.com/metabrainz/listenbrainz-ios/assets/106892217/e151a2b9-0cc0-42ab-ace7-4a670f3fa6de">
Ahh, it's giving different results, @akshaaatt

Reply:
@akshaaatt gotta use frame modifier for spacer instead of multiple spacers, I hope it's optimal than this.

Reply:
Works. Thanks